examples/market_making_strategies/strategies/naive_market_making/naive_market_making.py

```diff
@@ -0,0 +1,188 @@
+    self.order_size_per_side = 10
+    self.tick_time_seconds = tick_time_seconds
+
+  def float_round_down(self, probability: float, decimal: int = 2) -> float:
+    return floor(probability * (10**decimal)) / (10**decimal)
+
+  def read_probability_data(self, data: Dict):
+    self.order_size_per_side = int(data["orderSizePerSide"])
+    self.spread_size_dollars = float(data["spreadSizeDollars"])
+    self.tick_time_seconds = int(data["tickTimeSeconds"])
+    prob_list = data["probabilitiesLists"]
+    return [prob["longProbability"] for prob in prob_list if prob["injectiveMarketId"] == self.market_id]
+
+  def _get_prices(self):
+    pass
+
+  def get_prices(self, probabilities: List[float]) -> Tuple[List[float], List[float]]:
+    # TODO: maybe be its better with SortedList()
+    buy_long_prices = []
+    buy_short_prices = []
+
+    for probability in probabilities:
+      if probability < 1 - probability:
+        buy_long_prices.append(self.float_round_down(probability + self.spread_size_dollars / 2))
+        buy_short_prices.append(self.float_round_down(1 - probability - self.spread_size_dollars / 2))
+      else:
+        buy_long_prices.append(self.float_round_down(probability - self.spread_size_dollars / 2))
+        buy_short_prices.append(self.float_round_down(1 - probability + self.spread_size_dollars / 2))
+    buy_long_prices.sort()
+    buy_short_prices.sort(reverse=True)
+    return (buy_long_prices, buy_short_prices)
+
+  def _get_quantities(self, prices: List[float], multiplier) -> List[int]:
+    return [int(multiplier * price) for price in prices]
+
+  def get_quantities(self, prices: Tuple[List[float], List[float]]) -> Tuple[List[int], List[int]]:
+    buy_for_quantity = self._get_quantities(prices[0], self.order_size_per_side)
+    buy_against_quantity = self._get_quantities(prices[1], self.order_size_per_side)
+    return (buy_for_quantity, buy_against_quantity)
+
+  def get_new_order(self, prices: Tuple[List[float], List[float]], quantities: Tuple[List[int],
+                                                                                     List[int]]) -> List[Order]:
+    buy_for = self.build_buy_orders(prices=prices[0], quantities=quantities[0], sides=["long" for _ in quantities])
+    buy_against = self.build_buy_orders(prices=prices[1], quantities=quantities[1], sides=["short" for _ in quantities])
+    return buy_for + buy_against
+
+  def initilization(self):
+    self.fund_wallet()
+
+  def strategy(self):
+    while True:
+      sleep(self.tick_time_seconds)
+      if (len(self.buy_for) != 0):
+        self.buy_for.clear()
+      if (len(self.buy_against) != 0):
+        self.buy_against.clear()
+
+      # in here, we read this file very n seconds
+      json_file = open("examples/market_making_strategies/data/probabilities.json")
+      probabilities_data = json.load(json_file)
+      probabilities = self.read_probability_data(probabilities_data)
+      prices = self.get_prices(probabilities)
```

Comment:
if we're always going to be passing around two sets of prices, `buy_long` and `buy_short`, I think best practice would be to unpack the values rather than passing around the tuple itself.
So this would become
```python
buy_long_prices, buy_short_prices = self.get_prices(probabilities)
```
and other functions would need to be updated to account for this